### ISSUE TITLE
Enforce passing context to super in class constructor

### ISSUE THREAD

**Issue by miklschmidt** [secoya/tslint-secoya#4]
This is especially important for relay classes.

**Comment by kastermester** [comment]
We need to figure out if this always applies or not. I suspect something else was the actual bug here (but I'm not sure).

At the very least however, we should create a rule that enforces this when referencing `this.context` inside the react class.

**Comment by miklschmidt** [comment]
Yes, i think it actually wasn't the source of the problem. There's probably documentation somewhere that explains when it's necessary.
